plugins/inputs/nats_consumer/nats_consumer.go

```diff
@@ -205,6 +233,13 @@ func (n *natsConsumer) clean() {
 		}
 	}
 
+	for _, sub := range n.jsSubs {
+		if err := sub.Unsubscribe(); err != nil {
+			n.Log.Errorf("Error unsubscribing from subject %s in queue %s: %s",
+				sub.Subject, sub.Queue, err.Error())
```

Comment:
You can use
```suggestion
			n.Log.Errorf("Error unsubscribing from subject %s in queue %s: %v", sub.Subject, sub.Queue, err)
```
which is a bit shorter...